api/app/tests/Unit/Service/RestHandler/Report/DeputyCostsEstimateReportUpdateHandlerTest.php

```diff
@@ -8,25 +8,21 @@
 use App\Service\RestHandler\Report\DeputyCostsEstimateReportUpdateHandler;
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\ORM\EntityManager;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 
 class DeputyCostsEstimateReportUpdateHandlerTest extends TestCase
 {
-    /** @var DeputyCostsEstimateReportUpdateHandler */
-    private $sut;
-
-    /** @var EntityManager|\PHPUnit_Framework_MockObject_MockObject */
-    private $em;
-
-    /** @var Report|\PHPUnit_Framework_MockObject_MockObject */
-    private $report;
+    private DeputyCostsEstimateReportUpdateHandler $sut;
+    private EntityManager|MockObject $em;
```

Comment:
Is '&' better than '|', or are they interchangeable? I have seen '&MockObject' in more places in the code I think.